docker-compose.yml

```
@@ -24,6 +24,14 @@ services:
       - db
     volumes:
       - ./volumes/nextcloud:/var/www/html
+    environment:
+      - POSTGRES_DB=db
+      - POSTGRES_HOST=${POSTGRES_HOST:-db}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-SECRET_PASSWORD}
+      - POSTGRES_USER=${POSTGRES_USER:-nextcloud}
+      - NEXTCLOUD_ADMIN_USER=${NEXTCLOUD_ADMIN_USER:-admin}
+      - NEXTCLOUD_ADMIN_PASSWORD=${NEXTCLOUD_ADMIN_PASSWORD:-admin}
+      - NEXTCLOUD_TRUSTED_DOMAINS=${NEXTCLOUD_TRUSTED_DOMAINS:-mydomain.coop}  
```

If you don't define a value, the value will get from `.env` and de items of environment array will only to inform to docker-compose what environment is necessary to get value from `.env` and define

```suggestion
      - POSTGRES_DB
      - POSTGRES_HOST
      - POSTGRES_PASSWORD
      - POSTGRES_USER
      - NEXTCLOUD_ADMIN_USER
      - NEXTCLOUD_ADMIN_PASSWORD
      - NEXTCLOUD_TRUSTED_DOMAINS
```